Package.swift

```diff
@@ -21,51 +21,57 @@ let package = Package(
   products: [
     .library(
       name: "gRPC-Core",
-      targets: ["gRPC-Core"]
+      targets: ["grpc-Wrapper"]
     ),
     .library(
       name: "gRPC-C++",
-      targets: ["gRPC-CXX-Target"]
+      targets: ["grpcpp-Wrapper"]
     ),
   ],
   dependencies: [
     .package(
         url: "https://github.com/google/abseil-cpp-binary.git",
-        "1.2022062300.0" ..< "1.2022062400.0"
+        "1.2022062300.1" ..< "1.2022062400.0"
```

Comment:
@paulb777 , I pinned to dot 1, but maybe should pin to dot 0 to reduce surface area if a problem arises with abseil release. WDYT?

Reply:
I could go either way on this one. Pinning to .1 reduces the support matrix since all Firestore deps will get manifests versus seems unlikely abseil would go wrong separately from grpc.

Reply:
Okay, I'll keep it.